Update react-with-redux.md (#10300)
aspnet/Docs: aspnetcore/client-side/spa/react-with-redux.md

::: moniker range="= aspnetcore-2.0"

> [!NOTE]

> > This documentation isn't about the React-with-Redux project template included in ASP.NET Core 2.0. It's about the newer React-with-Redux template to which you can update manually. The template is included in ASP.NET Core 2.1 and later.

Reviewer: This predates the PR, but this lingo seems a bit too loose for APEX. :smile: lol

Also, "to which" phrasing is odd.

```suggestion
> This documentation doesn't pertain to the React-with-Redux project template included in ASP.NET Core 2.0. It pertains to the newer React-with-Redux template that you can update manually. The template is available in ASP.NET Core 2.1 or later.
```

::: moniker-end

The updated React-with-Redux project template provides a convenient starting point for ASP.NET Core apps using React, Redux, and [create-react-app](https://github.com/facebookincubator/create-react-app) (CRA) conventions to implement a rich, client-side user interface (UI).

With the exception of the project creation command, all information about the React-with-Redux template is the same as the React template. To create this project type, run `dotnet new reactredux` instead of `dotnet new react`. For more information about the functionality common to both React-based templates, see [React template documentation](xref:spa/react).

> For information on setting up a react-redux app set up as a sub-application in IIS, see [this github issue](https://github.com/aspnet/Templating/issues/555).

Reviewer: Suggestion: I try to avoid "this {anything}" link text. I try to use the title (or a modified title). Here's an option for this along those lines (includes a change to the prep phrase that leads it off) ...

```suggestion
For information on configuring a React-with-Redux sub-application in IIS, see [ReactRedux Template 2.1: Unable to use SPA on IIS (aspnet/Templating &num;555)](https://github.com/aspnet/Templating/issues/555).
```
